pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -49,59 +44,71 @@ social:
+                = hoc_s(:hoc_live_learn_more)
 
-  -# Off season interest form
-  -# This is only shown when hoc_mode is false
-  - if hoc_mode == false
-    %hr{style: "max-width: 960px; margin-inline: auto"}
-    %section
-      .wrapper
-        = view :"events/off_season_interest_form"
+  -# Host an Hour of Code
+  %section.no-padding-bottom
+    .wrapper
+      .flex-container.justify-space-between.align-items-center.flex-direction-column-tablet-reverse.gap-4.mobile-gap-2
+        .text-wrapper.flex-1
+          %h2.no-margin-top
+            =hoc_s("hoc_homepage.host.heading")
+          %p.body-two
+            =hoc_s("hoc_homepage.host.desc")
+          %a.link-button{href: resolve_url('/events')}
+            =hoc_s("hoc_homepage.host.button")
+        %figure.flex-1
+          %img.rounded-corners{src: "/images/homepage-students-computer.png", alt: "", style: "width: 100%"}
 
-  -# HoC Live Spanish banner
-  - if @language == 'es' || @language == 'la'
-    %section.spanish-banner.no-padding-bottom
-      .wrapper
-        .action-block.action-block--two-col
-          %img{src: '/images/hora-del-codigo-en-vivo.png', alt: hoc_s(:hoc_live)}
-          .content-wrapper
-            %h2.heading-lg
-              = hoc_s(:hoc_live_learn_title)
-            %p
-              = hoc_s(:hoc_live_learn_message)
-            %a.link-button{href: 'https://code.org/envivo'}
-              = hoc_s(:hoc_live_learn_more)
-        %hr
+  -# Resources
+  %section
+    .wrapper
+      .flex-container.justify-space-between.align-items-center.flex-direction-column-tablet.gap-4.mobile-gap-2
+        %figure.flex-1
+          %img.rounded-corners{src: "/images/homepage-resources.png", alt: "", style: "width: 100%"}
+        .text-wrapper.flex-1
+          %h2.no-margin-top
+            =hoc_s("hoc_homepage.resources.heading")
+          %p.body-two
+            =hoc_s("hoc_homepage.resources.desc")
+          .button-wrapper
+            %a.link-button{href: resolve_url('/how-to')}
+              =hoc_s("hoc_homepage.resources.button.how_to")
+            %a.link-button.secondary{href: resolve_url('/promote')}
+              =hoc_s("hoc_homepage.resources.button.explore_resources")
 
   -# Stats
-  %section.stats.bg-neutral-light
+  %section.stats.bg-neutral-dark
     .wrapper.centered
+      %h2.white
+        =hoc_s("hoc_homepage.global.heading")
+      %p.body-two.no-margin-bottom.white
+        =hoc_s("hoc_homepage.global.desc")
       = view :home_stats
-      .button-wrapper
-        %a.link-button{href: resolve_url('/how-to')}
+      .button-wrapper.centered
+        %a.link-button.white{href: resolve_url('/events')}
           =hoc_s(:call_to_action_host_hoc)
-        %a.link-button.secondary{href: resolve_url('/learn')}
+        %a.link-button.secondary.white{href: resolve_url('/learn')}
           =hoc_s(:call_to_action_try_activities)
 
   -# Organized by Code.org
   = view :organized_by_cdo
-
-:javascript
-
-  $(function() {
```

Comment:
This is leftover from the highlights being on this page which was deprecated here: https://github.com/code-dot-org/code-dot-org/pull/60623